doc/test-machine.md

```diff
@@ -81,10 +81,14 @@ conjunction with `with-open` to ensure that associated resources are shut down
 cleanly when you are finished with a machine.
 
 ```clojure
+(defn all-ok?
```

Comment:
actually I'm not too sure how this snippet of code is related with the above documentation about Lifecycle, maybe we could move it somewhere or give more context?